config.yaml

```diff
@@ -592,6 +592,7 @@ guilds:
         color: 0xEAC645
         hoist: true
   encounters:
+  # Changed initial cleared role colors
```

Comment:
typically when we add comments, we describe the functionality/context for the code that the comment is attached to. but this seems like a changelog, so we won't need this